src/content/en/updates/2018/10/animation-worklet.md

```diff
@@ -0,0 +1,493 @@
+especially important for AnimationWorklet.
+
+### Compositor NSync
+
+You might know that certain CSS properties are fast to animate, while others are
+not. Some properties just need some work on the GPU to be animated, while others
+force the browser to re-layout the entire document. Sites like
+[CSSTriggers.com](https://csstriggers.com/) show you which properties are fast
+to animate, and which are not.
+
+In Chrome (as in many other browsers) we have a process called the compositor,
+whose job it is — and I'm very much simplifying here — to arrange layers and
+textures and then utilize the GPU to update the screen as regularly as possible,
+ideally as fast as the screen can update (typically 60Hz). Depending on which
+CSS properties are being animated, the browser might just need have the
+compositor do it's work, while other properties need to run layout, which is an
+operation that only the main thread can do. Depending on which properties you
+are planning to animate, your animation worklet will either be bound to the main
+thread or run in a separate thread in sync with the compositor.
+
+Note: You should try avoiding "slow" properties at all costs. Limit yourself to
+animation `opacity` and `transform` to make sure your animations run smooth even
+on slow devices. Currently, Chrome's implementation doesn't even allow you to
+animate slow properties with AnimationWorklet and will throw a warning at you
+
+![DevTools showing an error when animating slow properts](/web/updates/images/2018/10/animation-worklet/slowprop.png)
+
+### Slap on the wrist
+
+There is usually only one compositor process which is potentially shared across
+multiple tabs, as the GPU is a highly-contended resource. If the compositor gets
+somehow blocked, the entire browser grinds to a halt and becomes unresponsive to
+user input. This needs to be avoided at all costs. So what happens if your
+worklet cannot deliver the data the compositor needs in time for the frame to be
+rendered?
+
+If this happens the worklet is allowed — per spec — to "slip". It falls behind
+the compositor, and the compositor is allowed to re-use the last frame's data to
+keep the frame rate up. Visually, this will look like jank, but the big
+difference is that the browser is still responsive to user input.
+
+Note: This is what the spec allows the browser to do. Chrome does not currently
+do any of these things, but will implement these behaviors soon™
+
+## Conclusion
+
+There are many facets to AnimationWorklet and the benefits it brings to the web.
+The obvious benefits are more control over animations and new ways to drive
+animations to bring a new level of visual fidelity to the web. But the APIs
+design also allows you to make your app more resilient to jank while getting
+access to all the new goodness at the same time.
+
+Animation Worklet is in Canary and we are aiming for an Origin Trial with Chrome
+69. We are eagerly awaiting you to build great new experiences on the web and
```

Comment:
Change to 'We are eagerly awaiting your great new web experiences and hearing about what we can improve.'